<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form</class>
 <widget class="QWidget" name="form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Signal Analyzer</string>
  </property>
  <widget class="QPushButton" name="_in_bt_measure">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Measure</string>
   </property>
  </widget>
  <widget class="QPushButton" name="_in_bt_save">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="_in_bt_analyse">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Analyse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>95</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Logging</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Serial number of device</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>81</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Serial number of sample</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="_in_le_sample_sn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="_in_pte_log">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>511</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="_in_le_device_sn">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>41</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="_in_bt_connect">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Connet</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
